spec/helpers/api_helper_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe ApiHelper, type: :controller do
+  include ApiHelper
+  it 'exportx organizations as markdown' do
+    create(:organization, name: 'markdowned', emails: ['email@foo'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.